.circleci/config.yml

```diff
@@ -428,6 +428,26 @@ jobs:
           paths:
             - .tox
 
+  influx:
+    docker:
+      - *test_runner
+      - image: influxdb:1.5-alpine
+    steps:
+      - checkout
+      - restore_cache:
```

Comment:
We centralized cache management just recently (😞 blame on me), so we know defined the `restore_cache` and `save_cache` steps in one place and then [we use anchors to](https://circleci.com/docs/2.0/writing-yaml/#anchors-and-aliases) to refer them. Please see here for an example here: https://github.com/DataDog/dd-trace-py/blob/master/.circleci/config.yml#L38

Reply:
Changed in b136a73d087d7dcd31a490335d6a3c7d3bfaf6a0